src/Tasks/Backup/BackupJob.php

```diff
@@ -227,7 +227,7 @@ protected function createZipContainingEveryFileInManifest(Manifest $manifest): s
 
         consoleOutput()->info("Created zip containing {$zip->count()} files and directories. Size is {$zip->humanReadableSize()}");
 
-        $this->sendNotification(new BackupZipWasCreated($pathToZip));
+        $this->encryptBackup($pathToZip);
```

Comment:
I don't understand why the function is named this way. It doesn't encrypt the backup, only sends a notification.

Reply:
https://github.com/spatie/laravel-backup/blob/daaf928f654731a8e9e9e1607ddd9c30a9a57d20/config/backup.php#L148-L149
Also if you create a `BackupZipWasCreatedNotification` class and add it to the config file, the notification will still be sent because the event `BackupZipWasCreated` is triggered, that makes `--disable-notifications` useless